db/databaseGenerator.js

```diff
@@ -0,0 +1,118 @@
+const mysql = require('mysql');
+const moment = require('moment');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '',
+});
+
+connection.connect();
+
+// Drop database if it exists:
+connection.query('drop database if exists expressel;', (error) => {
+  if (error) throw error;
+});
```

Comment:
Might be good to have some indicator that this completed successfully (console log message). Not required - just a suggestion

Reply:
Same with the rest of the queries